lib/data/replacements.json

```diff
@@ -48,8 +53,11 @@
       "replacements:vsts-task-lib-to-azure-pipelines-task-lib",
       "replacements:xmldom-to-scoped",
       "replacements:zap"
-    ],
-    "ignoreDeps": []
```

Comment:
Not sure this should be removed

Reply:
Should I add it back? I don't know what does this property mean.

Reply:
Yes, avoid removing things if you don't understand!

Reply:
@rarkins Done!